conf.yaml

```diff
@@ -81,6 +81,7 @@ contents:
             branches:   [ master ]
             tags:       Elastic Stack/Glossary
             subject:    Elastic Stack
+            asciidoctor: true
```

Comment:
One thought: Should/could this be a list of versions (akin to "branches") instead of a boolean? If so, it would give us more wiggle room if we can't convert some of the very old releases to asciidoctor builds.

Reply:
I'd be happy to do that when we figure out that we need it.